client/packages/electron/src/electron.ts

```diff
@@ -135,6 +136,16 @@ const start = (): void => {
   ipcMain.handle(IPC_MESSAGES.CONNECTED_SERVER, async () => connectedServer);
   ipcMain.handle(IPC_MESSAGES.START_BARCODE_SCAN, () => barcodeScanner.start());
   ipcMain.on(IPC_MESSAGES.STOP_BARCODE_SCAN, () => barcodeScanner.stop());
+  // preference method stubs: not currently implemented for electron
+  ipcMain.on(IPC_MESSAGES.GET_PREFERENCE, (_event, preference: Preference) => ({
```

Comment:
am thinking to implement with `electron-store` but thought to make a new issue for that